### PR TITLE
Update model.py

### PR DESCRIPTION
**Pull request by amallia**

### CODE REVIEW

File: src/model.py [hunk 1]
```diff
@@ -21,10 +21,14 @@ def __init__(self, config):
         self.regex_multi_space = re.compile('\s+')
```
Comment (tonellotto) [inline src/model.py]: Remove regexes

File: src/model.py [hunk 2]
```diff
@@ -21,10 +21,14 @@ def __init__(self, config):
...
-
+        
+        self.impact_score_encoder = nn.Sequential(
+            nn.Linear(config.hidden_size, config.hidden_size),
+            nn.ReLU(),
+            nn.Dropout(config.hidden_dropout_prob),
+            nn.Linear(config.hidden_size, 1),
+            nn.ReLU()
+        )
         self.init_weights()
```
Comment (tonellotto) [inline src/model.py:27]: Space

File: src/model.py [hunk 3]
```diff
@@ -114,12 +118,7 @@ def one(i):
...
 
-        pooled_output = self.pre_classifier(pooled_output)
-        pooled_output = nn.ReLU()(pooled_output)
-        pooled_output = self.dropout(pooled_output)
-
-        y_score = self.classifier(pooled_output)
-        y_score = torch.nn.functional.relu(y_score)
+        y_score = self.impact_score_encoder(pooled_output)
 
         x = torch.arange(bsize).expand(len(pfx_sum), bsize) < torch.tensor(pfx_sum).unsqueeze(1)
```
Comment (tonellotto) [inline src/model.py:117]: These masks are metaphysicaloric!

Comment (amallia) [reply src/model.py:117]: Will fix separately
